vertx-sql-client/src/test/java/io/vertx/sqlclient/tck/PreparedQueryTestBase.java

```diff
@@ -138,6 +138,18 @@ public void testPreparedUpdate(TestContext ctx) {
     }));
   }
 
+  @Test
+  public void testPreparedUpdateWithoutChange(TestContext ctx) {
```

Comment:
I don't think we should add this to the TCK, because different database servers by default behave differently to return a rowcount and they are not bound to return the found rows here.